streaming-pacman/config/demo.cfg.example

```diff
@@ -1,7 +0,0 @@
-export AWS_ACCESS_KEY="<AWS_ACCESS_KEY>"
```

Comment:
Hi @jcrabb-confluent , we need this file to stay in the repo, otherwise people won't know how the config file looks like :) Can you add the file again? Thanks!